src/Meshes/mesh.ts

```diff
@@ -799,17 +812,17 @@ export class Mesh extends AbstractMesh implements IGetSetVerticesData {
     /**
      * Add a mesh as LOD level triggered at the given distance.
      * @see https://doc.babylonjs.com/how_to/how_to_use_lod
-     * @param distance The distance from the center of the object to show this level
+     * @param distanceOrScreenCoverage Either distance from the center of the object to show this level or the screen coverage if `useScreenCoverage` is set to `true`

```

Comment:
nit: May be add `(value between 0 and 1)` to make clear the screen coverage value is a percentage

Reply:
it is ok as this is not binding for either TS or JS